src/index.css

```diff
@@ -8,6 +12,28 @@ html {
     margin: 0;
     padding: 0;
     box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+
+@layer base {
+    body {
+        @apply h-full bg-pureWhite;
+    }
+}
+
+@layer components {
```

Comment:
Hi @JoHeyJo! Can you please clarify, what is the better approach: to use the reusable component with inline styles for different button sizes, colors etc, where we will send colors, sizes, other staff via props, or using @layer in css? because this seems totally new approach that interfere with inline styles. And should be consider as global refactor. I do not see the benefit of using Tailwind if we will use css file for styles. Seems Tailwind recommend [Avoiding premature abstraction](https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction)

Reply:
Yeah, using `layers` or other css files will defeat the purpose of Tailwinds utility class approach to styling. First, rely on Tailwinds inline styles unless necessary at which point you can use props for dynamic styling.